tools/new-component/templates/demo-pattern.twig

```diff
@@ -1 +1,9 @@
-{% include '@<%= cleanPatternType %>/_<%= name %>.twig' %}
+{% embed '@protons/demo/demo-block.twig' with {
```

Comment:
Can we use `extends` here instead to:

1. Avoid indentation and nesting
1. Indicate that this is a perfect use-case for extending

Reply:
Note: we can set vars in extends.

Reply:
Updated!